packages/manager/modules/nutanix/src/dashboard/routing.js

```diff
@@ -41,6 +42,12 @@ export default /* @ngInject */ ($stateProvider) => {
       technicalDetails: /* ngInject */ (getTechnicalDetails) =>
         getTechnicalDetails.baremetalServers,
       breadcrumb: /* @ngInject */ (serviceName) => serviceName,
+      expressOrderLink: /* @ngInject */ (coreConfig) => {
+        const urls = getConstants(coreConfig.getRegion()).URLS;
+        console.log({ urls, constant: getConstants(coreConfig.getRegion()) });
+        return urls[coreConfig.getUser().ovhSubsidiary ?? urls.FR]
```

Comment:
By the way, urls.FR could also be null for region CA,US so you should remove it too
```suggestion
        return urls[coreConfig.getUser().ovhSubsidiary] ?? urls.FR
```